Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: default
+
+default: pull-request-ci
+
+pipeline-build-amd64:
+	cargo build --release --features bpf \
+		&& ./package.sh amd64/rezolus.tar.gz target/release/rezolus \
+
+pipeline-build-arm64v8:
+	/sbin/bpftool btf dump file /sys/kernel/btf/vmlinux format c > src/common/bpf/vmlinux.h \
+		&& cargo build --release --features bpf \
+		&& ./package.sh arm64v8/rezolus.tar.gz target/release/rezolus \
```

Comment:
`package.sh` appears to be missing contents 😅

Reply:
🤦  so embarrassed. Fixed now https://github.com/momentohq/rezolus/pull/1/commits/4ef393eeb0b49264e47ff3839cc243e084040875